markups/restructuredtext.py

```diff
@@ -15,7 +15,7 @@
 else:
 	class CustomHTMLTranslator(HTMLTranslator):
 		def starttag(self, node, tagname, suffix='\n', empty=False, **attributes):
-			if node.line is not None:
+			if (node is not None) and hasattr(node, 'line') and (node.line is not None):
```

Comment:
Can this be simplified a bit?

```suggestion
			if node is not None and getattr(node, 'line', None) is not None:
```